components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -6820,24 +6891,14 @@ func insertTestProject(
 	id string,
 	name string,
 	projType storage.Type) storage.Project {
-	return insertTestProjectWithRules(t, db, id, name, projType, storage.NoRules)
-}
-
-func insertTestProjectWithRules(
-	t *testing.T,
-	db *testhelpers.TestDB,
-	id string,
-	name string,
-	projType storage.Type,
-	status storage.ProjectRulesStatus) storage.Project {
 	t.Helper()
-	proj, err := storage.NewProject(id, name, projType, status)
+	// the status is not actually used. we are only using NewProject for the id
+	// and name validation and to have a storage.Project object to return.
+	proj, err := storage.NewProject(id, name, projType, storage.NoRules)
```

Comment:
Not critical, but perhaps we pull `storage.NoRules.String()` into a constant like `DEFAULT_RULE_STATUS` for use throughout this file. That might make it slightly less obtuse when used in this helper.